### DISCUSSION TITLE
Add support for additional APIs of Azure OpenAI and Anyscale providers

### DISCUSSION THREAD

**Discussion by the-gigi** [sashirestela/simple-openai#103] [Ideas]
simple-openai currently throws "Not implemented" exceptions for most endpoints for these providers. However, it is actually very likely that they support additional APIs (with the proper models). They were just never tested.

I suggest to create Azure OpenAI and Anyscale versions for all the demos. Then evaluate, which endpoints are actually supported and with what restrictions (e.g. currently streaming is not supported for Azure OpenAI).

**Comment by sashirestela** [comment]
Check the updated structure of demos
